src/TileChartMap.jsx

```diff
@@ -0,0 +1,54 @@
+import './TileChartMap.less'
+import { connect } from 'react-redux'
+import React from 'react'
+import { TileMap } from 'cfpb-chart-builder'
+
+export class TileChartMap extends React.Component {
+  componentDidUpdate( prevProps ) {
+    const props = this.props
+    if ( !props.data[0].length ) {
+      return
+    }
+
+    if ( JSON.stringify( prevProps.data ) !== JSON.stringify( props.data ) ) {
+      this._redrawMap()
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <div id="tile-chart-map"
+             className="cfpb-chart"
+             data-chart-type="tile_map">
+        </div>
+      </div>
+    )
+  }
+
+  // --------------------------------------------------------------------------
+  // Event Handlers
+  _redrawMap() {
+    const colors = [
+      'rgba(247, 248, 249, 0.5)',
+      'rgba(212, 231, 230, 0.5)',
+      'rgba(180, 210, 209, 0.5)',
+      'rgba(137, 182, 181, 0.5)',
+      'rgba(86, 149, 148, 0.5)',
+      'rgba(37, 116, 115, 0.5)'
+    ]
+
+    // eslint-disable-next-line no-unused-vars
+    const chart = new TileMap( {
+      el: document.getElementById( 'tile-chart-map' ),
+      data: this.props.data,
+      colors,
+      localize: true
+    } )
+  }
```

Comment:
I'm assuming `chart` is not used because `new TileMap` embeds itself into the DOM at `document.getElementById( 'tile-chart-map' )`

Assigning it to `const _` might get rid of the warning

Reply:
I tried that and it's still nagging me about the error.    Chart isn't used, it's just how the constructor works.   I can remove the const chart =  and do this instead

https://stackoverflow.com/questions/33127663/pattern-for-javascript-instantiation-which-doesnt-upset-eslint

But I'm throwing in another eslint ignore message

Reply:
It's fine as is, these edge cases are annoying to try and stay compliant with